Classes/Report/AccessFilterPluginInstalledStatus.php

```diff
@@ -63,7 +65,7 @@ class AccessFilterPluginInstalledStatus implements StatusProviderInterface
     public function getStatus()
     {
         $reports = array();
-        $solrConnections = GeneralUtility::makeInstance('ApacheSolrForTypo3\\Solr\\ConnectionManager')->getAllConnections();
+        $solrConnections = GeneralUtility::makeInstance(ConnectionManager::class)->getAllConnections();
```

Comment:
This and other `::class` should probably not be part of the PR - although I agree they should be used!